src/app/db/db_StageStore.py

```
@@ -0,0 +1,179 @@
+        """Replace all stages for a task with the provided mapping.
+
+        Parameters:
+            task_id (str): Task identifier whose stages should be reset.
+            stages (dict[str, dict]): Mapping of stage name to metadata.
+
+        Side Effects:
+            Deletes existing rows in ``task_stages`` for the task and re-inserts
+            the provided entries in bulk.
+        """
+
+        now = self._current_ts()
```

Pylint (4.0.1)

[error] Instance of 'StageStore' has no '_current_ts' member

(E1101)